src/routes/safe/components/Balances/Collectibles/index.tsx

```diff
@@ -95,46 +97,69 @@ const Collectibles = (): React.ReactElement => {
     setSendNFTsModalOpen(true)
   }
 
-  return (
-    <Card className={classes.cardOuter}>
-      <div className={classes.cardInner}>
-        {/* No collectibles */}
-        {nftAssetsFromNftTokens.length === 0 && (
+  if (nftAssetsFromNftTokens.length === 0) {
```

Comment:
Can we also add a loading state? Currently it shows "No collectibles available" while loading.

Reply:
I'm going create a separate issue for this as it is a bigger task than it seems. It'll be easier to review.

Reply:
Fixed with #3701